### ISSUE TITLE
Decprecated code?

### ISSUE THREAD

**Issue by Dylan-Chapman** [ileri/postcss-bem#10]
I'm using gulp-postcss@6.0.1 and when I compile a file, I get the warning:

```
Container#eachAtRule is deprecated. Use Container#walkAtRules instead.
Node#removeSelf is deprecated. Use Node#remove.
```

I tested simply replacing all instances of ```eachAtRule``` with ```walkAtRules``` and replacing all instances of ```removeSelf``` with ```remove``` and it seemed to work. However, I don't actually know if other code needs to be changed or if there are any caveats with simply replacing these.

**Comment by Volosojui** [comment]
I have the same problem.

**Comment by ileri** [comment]
Fixed by #11.
